src/country_workspace/contrib/hope/constants.py

```diff
@@ -13,3 +13,13 @@
 ADMINAREAS_FIELDSET_NAME: Final[str] = "HOPE Admin Areas"
 ACCOUNT_FIELDSET_NAME: Final[str] = "HOPE Account"
 DOCUMENT_FIELDSET_NAME: Final[str] = "HOPE Document"
+
+DOCUMENT_PREFIXES_TO_TYPE_MAPPING = {
+    "national_id__": "national_id",
+    "national_passport__": "national_passport",
```

Comment:
let's change the double __ with a single _. Since the XLSX template is using just one.